src/main/resources/db/migration/0200/V226__OrganizationTypes.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE managed_facility_types (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE organization_types (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE organization_managed_facility_types (
+  organization_id BIGINT NOT NULL REFERENCES organizations,
+  managed_facility_type_id INTEGER NOT NULL REFERENCES managed_facility_types,
+  PRIMARY KEY (organization_id, managed_facility_type_id)
+);
+
+ALTER TABLE organizations ADD COLUMN type_id INTEGER REFERENCES organization_types;
+ALTER TABLE organizations ADD COLUMN type_details VARCHAR(100);
+ALTER TABLE organizations ADD COLUMN website TEXT;
+
+/* Allow type details to be populated only if type id is 'Other' (6) */
+ALTER TABLE organizations ADD CONSTRAINT other_type_details CHECK (type_details IS NULL OR (type_details <> '' AND type_id = 6));
```

Comment:
Does this check fail if `type_details = ''`, regardless of the value for `type_id`?

Reply:
Should it instead be `CHECK (type_details IS NULL OR (type_details IS NOT NULL AND type_id = 6));`?

Reply:
Oh, maybe it should be `CHECK (type_details IS NULL OR type_id = 6);`

Reply:
we want a non-empty value for type details if type id is 'Other'

Reply:
Oh, I see. It looks like the constraint as written should allow for `type_details` to be NULL and `type_id = 6` at the same time.

Reply:
I will update the comment.

Reply:
> Oh, I see. It looks like the constraint as written should allow for `type_details` to be NULL and `type_id = 6` at the same time.

Ugh, yes, good catch - will fix that. Thanks!!